osu.Game/Screens/Edit/Compose/Components/SelectionBoxScaleHandle.cs

```diff
@@ -50,14 +50,14 @@ protected override void OnDrag(DragEvent e)
 
             rawScale = convertDragEventToScaleMultiplier(e);
 
-            applyScale(shouldLockAspectRatio: isCornerAnchor(originalAnchor) && e.ShiftPressed);
+            applyScale(shouldLockAspectRatio: isCornerAnchor(originalAnchor) && e.ShiftPressed, ignoreAnchor: e.AltPressed);
```

Comment:
<kbd>Alt</kbd> is not the conventional choice for this behaviour. In gimp this is under <kbd>Ctrl</kbd>. And I believe it's the same in photoshop (although I don't have a copy to check right now).

I'd probably prefer this was <kbd>Ctrl</kbd>. Is there a particular reason for choosing <kbd>Alt</kbd>?

Reply:
I used `Alt` because from my experience it is the typical modifier key used for this (photoshop, figma, indesign, krita all use the `Alt` key), but after some research I found some programs that use `Ctrl` too (affinity photo, microsoft word), so I'm open to change it. `Alt` is definitely the first key I tried out when I was originally looking for this feature though.

Reply:
Oh figma does use <kbd>Alt</kbd>... interesting.

I did test https://www.photopea.com/ which is a sort of photoshop clone and that used <kbd>Ctrl</kbd> so I extrapolated photoshop did the same, although possibly too rashly.

I'm not sure I'm hugely fussed as to which one it is in the end, but it'd be good to know if one is more frequently used than the other...

Reply:
I can compile a list of what modifier is used for different programs, however I'm a bit unsure which programs to check here since the original list already includes pretty much every program that I could think of off the top of my head (apart from checking the entire adobe/affinity suite I guess).

Reply:
Nah it's fine. I'm ok with just going with alt for now. If figma and adobe use that then that's probably the one users will most expect.